textile/chat-features.textile

```diff
@@ -104,121 +104,144 @@ The status of any given Chat Room is the combination of the states of all of its
+*** @(CHA-RL1k3)@ @[Testable]@ If the attachment is unsuccessful (i.e. @channel.attach()@ throws an error), the attach error will be thrown as the result of the room @attach()@ operation.
+** @(CHA-RL1f)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL1g)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL1g1)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL1g2)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL1g3)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL1h)@ This specification point has been removed. It was valid up until the single-channel migration.
 *** @(CHA-RL1h1)@ This clause has been deleted.
-*** @(CHA-RL1h2)@ @[Testable]@ If the underlying Realtime Channel entered the @SUSPENDED@ state (i.e. the @attach()@ operation fails and upon subsequently checking the channel state, it is @SUSPENDED@), then the room status will transition to @SUSPENDED@. The state transition @ErrorInfo@ will use the error from the realtime channels `attach()` as the @cause@. The status code shall be @500@ and the error code whatever attachment error code corresponds to the feature that has failed, per "the error code list":#error-codes.
-*** @(CHA-RL1h3)@ @[Testable]@ When the room enters the @SUSPENDED@ status as a result of @CHA-RL1h2@ the @attach()@ operation shall fail using the error from @CHA-RL1h2@. The room lifecycle shall asynchronously (non-blocking to the @ATTACH@ operation) enter the recovery loop (per "@CHA-RL5@":#CHA-RL5) and attempt to re-establish attachment.
-*** @(CHA-RL1h4)@ @[Testable]@ If the underlying Realtime Channel entered the @FAILED@ state, then the room status will transition to @FAILED@, using the error from the realtime channels `attach()` call as the @cause@ field of the @ErrorInfo@. The status code shall be @500@ and the error code whatever attachment error code corresponds to the feature that has failed, per "the error code list":#error-codes. The same error shall be thrown as the result of the @ATTACH@ operation.
-*** @(CHA-RL1h5)@ @[Testable]@ When the room enters the @FAILED@ status as a result of @CHA-RL1h4@, asynchronously with respect to @CHA-RL1h4@, then the room will detach all channels that are not in the @FAILED@ state.
-*** @(CHA-RL1h6)@ @[Testable]@ If, when performing @CHA-RL1h5@, a channel fails to detach on command, then the detach operation will be repeated until such a time that all channels have detached successfully.
-** @(CHA-RL1i)@ This specification point has been removed. It was superseded by CHA-RC1f.
```

Comment:
I don't think the CHA-RL1i tombstone should have been removed here?

Reply:
Re-added!